docs/association-financials.md

```diff
@@ -4,17 +4,19 @@ outline: deep
 
 # Wallets and Contracts controled by the association
```

Comment:
I think the title needs to be more general, e.g. "Association Financials". The Wallets and Contracts section can be a top-level heading and the Station Rewards a subheading.

Reply:
Can you please push the PR if wording optimization is the only problem? I want to submit details about the circulating supply to Coinmarketcap and Coingecko, and we've been iterating on the wording of the page for three days.

Reply:
Happy to push once the text is corrected.

Reply:
@eeVoskos Update the text so that it satisfies your specifications, and let me know pls.